game.js

```diff
@@ -0,0 +1,359 @@
+            pressbtn(5);
+        } else if (currbtn === 5){
+            pressbtn(3);
+        } else {
+            pressbtn(5);
+        }
+    } else {
+        if (bwin() === 0) {
+            if (hwin() === 0) {
+                var i = 0;
+                while (i < 9) {
+        if (clicked[i] === 0) {
+            pressbtn(i+1);
+            break;
+            }
+            i += 1;
+        }
+        
+            } else {
+                pressbtn(hwin());
+            }
+        } else {
+            pressbtn(bwin());
+        }
+    }
+    hsteps += 1;
+}
+function botmove() {
+    if (bfirst === 1) {
+        if (bsteps === 0) {
+            bdecider = currbtn;
+            pressbtn(bmap1.get(bdecider));
+        } else if (bsteps === 1) {
+            if (clicked[bmap2.get(bdecider) - 1] === 0) {
+                pressbtn(bmap2.get(bdecider));
+            } else {
+                pressbtn(bmap3.get(bdecider));
+            }
+        } else if (bsteps === 2) {
+            if (clicked[bmap4.get(bdecider) - 1] === 0) {
+                pressbtn(bmap4.get(bdecider));
+            } else {
+                pressbtn(bmap5.get(bdecider));
+            }
+        } else if (bsteps === 3) {
+            if (clicked[bmap6.get(bdecider) - 1] === 0) {
+                pressbtn(bmap6.get(bdecider));
+            } else {
+                pressbtn(bmap7.get(bdecider));
+            }
+        } else if (bsteps === 4) {
+            pressbtn(bmap7.get(bdecider));
+        }
+        bsteps += 1;
+    } else {
+        hmove();
+        //Human First Code
+    }
+}
+
+
+
+
+
+/* ------------------------------------------------------------------------------------------------------------- */
+function solve() {
+if (check() === 0) {
+if (twop === 0 ) {
+//Main Code
+
+} else {
+    if (turn % 2 === 1 ) {
+        currentm.classList.add('hide');
+        currentm = document.querySelector('.p2turn');
+        currentm.classList.remove('hide');
+    } else {
+        currentm.classList.add('hide');
+        currentm = document.querySelector('.p1turn');
+        currentm.classList.remove('hide');
+    }
+    //Switch turn messeges
+}
+} else if (check() === 1) {
+//Win
+if (twop === 1) {
+currentm.classList.add('hide');
+over = 1;
+if (whowin === 2) {
+winp = document.querySelector('.p1win');
+winp.classList.remove('hide');
+} else {
+winp = document.querySelector('.p2win');
+winp.classList.remove('hide');
+}
+} else {
+over = 1;
+currentm.classList.add('hide');
+var winm = document.querySelector('.win');
+winm.classList.remove('hide');
+}
+} else {
+//Draw
+over = 1;
+currentm.classList.add('hide');
+var drawm = document.querySelector('.draw');
+drawm.classList.remove('hide');
+}
+}
+
+
+/* --------------------------------------------------------------------------------------------------------------------------------*/
+document.querySelector('.b1').onclick = function() {
```

Comment:
You could've used DOM array and use `addEventListener` and each element could have ID which refers to the number in the grid.